concourse/pipelines/build-images.yml

```
@@ -0,0 +1,89 @@
+---
+definitions:
+
+resources:
+  - icon: github
+    name: smokey-commit
```

I think it's nice if the name comes first in these blocks (even thought it's equivalent).

I wonder if `smokey` would be just as good a name as `smokey-commit` (I wonder about the `-commit` suffix, as it's not really the _commit_ you're fetching, you're getting the whole repo. So `smokey-repo`? But I think just `smokey` is enough. We should probably have a styleguide / linter 😅 )

```suggestion
  - name: smokey-commit
    icon: github
```